### PR TITLE
refactor!: use gatbys SSR and Browser APIs to handle common Layout components

### PR DESCRIPTION
**Pull request by hollandjg**
Breaking Change: Requires updating the `path` in the Navbar config to have starting and ending slashes `/`

### CODE REVIEW

File: packages/gatsby-theme-project-portal/src/layouts/ProjectDetailPage.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const ProjectDetailPage: FunctionComponent<
   } = props
 
   return (
-    <Layout title={project.title} description={project.summary}>
```
Comment (jashlu) [inline packages/gatsby-theme-project-portal/src/layouts/ProjectDetailPage.tsx:31]: Quick question, so what happens to these descriptions and title?

**Review by jashlu — APPROVED** [review]

File: packages/gatsby-theme-project-portal/src/layouts/AboutPageLayout.tsx [hunk 2]
```diff
@@ -54,11 +54,7 @@ export const AboutPageLayout: FunctionComponent<AboutProps> = ({
   },
 }: AboutProps) => {
   return (
-    <Layout
-      activePage="About"
-      title="About"
-      description={`About the ${siteTitle}`}
-    >
+    <>
```
Comment (jashlu) [inline packages/gatsby-theme-project-portal/src/layouts/AboutPageLayout.tsx:57]: There are a couple of the import statements that can be cleaned up, including `Layout`

File: packages/gatsby-theme-project-portal/src/layouts/ProjectDetailPage.tsx (hunk 1 above)
Comment (jashlu) [reply packages/gatsby-theme-project-portal/src/layouts/ProjectDetailPage.tsx:31]: Looking at it overall, doesnt seem like we need title and description anymore right? Should we clean it up

Comment (hollandjg) [reply packages/gatsby-theme-project-portal/src/layouts/ProjectDetailPage.tsx:31]: The Layout props title and description are passed to the Head component and merged into the metadata, which is no longer handled by the Layout. I've taken out the extra props on the Layout components – thanks for catching this!

File: packages/gatsby-theme-project-portal/src/layouts/AboutPageLayout.tsx (hunk 2 above)
Comment (hollandjg) [reply packages/gatsby-theme-project-portal/src/layouts/AboutPageLayout.tsx:50]: Good catch, thanks!

**Review by jashlu — APPROVED** [review]
Confirming, looks good!

**Pull request merged**